target/linux/bmips/patches-5.15/200-mips-bmips-automatically-detect-CPU-frequency.patch

```diff
@@ -58,17 +58,19 @@ Signed-off-by: Álvaro Fernández Rojas <noltari@gmail.com>
  
  extern bool bmips_rac_flush_disable;
  
+@@ -40,6 +74,11 @@ extern bool bmips_rac_flush_disable;
+ 
  static const unsigned long kbase = VMLINUX_LOAD_ADDRESS & 0xfff00000;
  
 +struct bmips_cpufreq {
 +	const char		*compatible;
 +	u32			(*cpu_freq)(void);
-+};
++}
```

Comment:
This change looks wrong to me. 
Why did your remove the `;`?

Reply:
I see now that removing the other patch mentioned by @dragonbluep and refreshing did not need modifications to this patch at all.